gapipy/query.py

```diff
@@ -74,7 +96,7 @@ def get(self, resource_id, variation_id=None, cached=True, headers=None,
         something Falsey as `httperrors_mapped_to_none` like a `None` or an
         empty list.
         """
-        key = self.query_key(resource_id, variation_id)
+        key = self.query_key(resource_id, variation_id) # noqa
```

Comment:
The flake8 warning for this line is `local variable 'key' is assigned to but never used `. We should just delete the line instead of silencing the warning.